### PR TITLE
refactoring of example folder

### PR DESCRIPTION
**Pull request by RustamGimadiev**
- added inner examples structure
- all examples are simple Terraform code, not modules

### CODE REVIEW

File: examples/common/providers.tf [hunk 1]
```diff
@@ -4,7 +4,7 @@ terraform {
 
 provider "aws" {
   version = "2.66"
-  region  = var.aws_region
+  region  = "us-west-2"
```
Comment (akastav) [inline examples/common/providers.tf:7]: Could you please add notation about this params to readme

File: QUICKSTART.md [hunk 2]
```diff
@@ -168,37 +159,32 @@ The Swiss Army Kube repository has three main directories that provide a minimal
...
 * `charts`  - local Helm repository for Helm charts that can't be retrieved from public repositories.
 * `docs`    - more detailed documentation and various FAQ's
-* `example` - directory to be used as a template for your projects that includes configuration files for modules and variables.
+* `examples` - directories with different types of SAK use-cases to be used as a template for your projects that includes configuration files for modules and variables.
 * `modules` - Terraform modules (must-have and optional) to deploy your cluster with.
 
 ### Project Structure (Example Directory) 
 
-The `swiss-army-kube/example` directory is a boilerplate for your projects. Use it as a template. You can rename it to your project name for convenience. Make as many projects as you need by cloning and modifying this directory.  
+The `swiss-army-kube/examples` directory are a boilerplates for your projects, use them as a template. You can rename it to your project name for convenience. Make as many projects as you need by cloning and modifying this directory.
```
Comment (Evanto) [inline QUICKSTART.md:167]: The `swiss-army-kube/examples` directory contains project examples that you can use as boilerplates to start your new projects. Pick one, rename it to your project name for convenience, and modify the directory as required. This way you can create as many projects as you need really fast.

File: QUICKSTART.md [hunk 3]
```diff
@@ -168,37 +159,32 @@ The Swiss Army Kube repository has three main directories that provide a minimal
...
 
 1. Add an IAM user in AWS Console for developer with programmatic access
-2. Add user ARN and name to `user_arns` variable with group `system:developers` in `terraform.tfvars` file
+2. Add user ARN and name to `user_arns` variable with group `system:developers` for _kubernetes_  module in `main.tf` file
 3. Run `terraform plan -out=plan` and review
 4. Run `terraform apply plan`
 5. Send `kubeconfig_internal-projects` config and IAM user tokens to the developer
 
-NOTE: To change developers permissions on Kubernetes cluster manipulate `cluster_roles` variable in the `terraform.tfvars` file.
+NOTE: To change developers permissions on Kubernetes cluster manipulate `cluster_roles` variable in the `main.tf` file.
```
Comment (Evanto) [inline QUICKSTART.md:187]: NOTE: To change developers' permissions on the Kubernetes cluster edit the `cluster_roles` variable in the `main.tf` file.

File: README.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ Swiss Army Kube (SAK) is an open-source IaC (Infrastructure as Code) collection
 
 Swiss Army Kube uses Terraform to describe the desired state of your infrastructure (resources that need to be provisioned like IAM roles, ASG, Route 53, subnets, etc.) and build a Kubernetes cluster on AWS EC2 instances.   
 
-SAK provides an example directory that you can use as an easily modifiable template to set up your cluster deployment configuration in minutes. All you need is to edit a couple of files to include modules and set variables. This way you can quickly configure and provision multiple dedicated EKS Kubernetes clusters with different configurations of modules, variables, networks, and Kubernetes versions.
+SAK provides an examples directories that you can use as an easily modifiable template to set up your cluster deployment configuration in minutes. All you need is to edit a couple of files to include modules and set variables. This way you can quickly configure and provision multiple dedicated EKS Kubernetes clusters with different configurations of modules, variables, networks, and Kubernetes versions.
```
Comment (Evanto) [inline README.md:19]: SAK provides example directories that you can use as easily modifiable templates to set up your cluster deployment configuration in minutes. All you need is to edit a couple of files to include modules and set variables. This way you can quickly configure and provision multiple dedicated EKS Kubernetes clusters with different configurations of modules, variables, networks, and Kubernetes versions.

File: README.md [hunk 5]
```diff
@@ -58,18 +58,16 @@ Configure and deploy as many projects as you want.
 2. Install Prerequisites
    + Clone this repository
    + Install prerequisites via script (MacOS users) or manually (other users)
-   + Rename `example.tfvars` to `terraform.tfvars`
-3. Configure your EKS cluster deployment using example directory as a project template
+3. Configure your EKS cluster deployment using one of the `examples/` directories as a project template
    + Configure modules
```
Comment (Evanto) [inline README.md:62]: maybe modules and variables?

**Review by Evanto — CHANGES_REQUESTED** [review]

**Review by akastav — APPROVED** [review]

**Review by Evanto — DISMISSED** [review]

**Review by akastav — APPROVED** [review]

**Pull request merged**